pkg/controller/garbagecollector/dump.go

```diff
@@ -273,6 +273,8 @@ func (h *debugHTTPHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
```

Comment:
what is the mime-type of this data?

Reply:
It looks like it's graphviz code. Actually, there is a `text/vnd.graphviz` type reserved in iana, but I don't know exactly what the implications of that are.

Reply:
yeah, I'd change this to `text/vnd.graphviz`

Reply:
Done.